content/zh/docs/concepts/overview/components.md

```diff
@@ -22,17 +22,6 @@ card:
   weight: 20
 -->
 
-<!--
-When you deploy Kubernetes, you get a cluster.
-{{< glossary_definition term_id="cluster" length="all" prepend="A Kubernetes cluster consists of">}}
```

Comment:
Do not remove this paragraph, instead, change line 27 to :

```
{{</* glossary_definition term_id="cluster" length="all" prepend="A Kubernetes cluster consists of" */>}}
```

Reply:
Please also help fix line 195-196 where the word "Kubernetes" is broken

Reply:
done